src/configuration/arguments-configuration-source.ts

```diff
@@ -66,6 +73,7 @@ export function getArgumentConfiguration(provider: ModProviderName): PartialConf
     const bindings: { [Key in keyof Arguments]: PartialConfiguration } = {
         "modpack-id": { "modpacks.ch": { modpack: { id: args["modpack-id"] } } },
         "modpack-version": { "modpacks.ch": { modpack: { version: args["modpack-version"] } } },
+        "manifest-file": { curseforge: { manifestFile: args["manifest-file"] ?? "manifest.json" } }, // Couldn't find a way to set default in a nicer way

```

Comment:
It does not need a default value here. The default value can be specified in `default-configuration-source.ts`. That value will be used if no other source sets it.